### PR TITLE
deps: update to `bpmn-js-element-templates@1.9.2`

### PR DESCRIPTION
**Pull request by marstamm**
Would be cool to get this into the 5.18 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+### Key Changes in Element Templates
```
Comment (smbea) [inline CHANGELOG.md:9]: Might as well add the version now so it can be released later 🙂

Comment (marstamm) [reply CHANGELOG.md:9]: I was not sure if there is anything else outstanding (so it could either be patch/minor release)

Comment (marstamm) [reply CHANGELOG.md:9]: Seems like we are good to go on other fronts, so I'll add a version number

Comment (smbea) [reply CHANGELOG.md]: Maybe we could also include the ````DEPS`: update to `bpmn-js-element-templates@1.9.1```` portion in the changes

**Review by smbea — APPROVED** [review]
Left a small comment but good to go otherwise

**Comment by marstamm** [comment]
Thanks, I changed the Changelog and will go ahead and release 🚀

**Pull request merged**